common/src/dodeca.rs

```diff
@@ -346,4 +346,15 @@ mod tests {
             epsilon = 1e-10
         );
     }
+
+    #[test]
+    fn dual_to_chunk_factor_correct() {
```

Comment:
This should be included in the commit that introduced the factor.

Reply:
I decided to replace this test with easier-to-read tests that check that the dodecahedron's origin is in the right place in chunk coordinates. This tests `Vertex::dual_to_chunk_factor()` indirectly.